### DISCUSSION TITLE
is there any simmilar exploit

### DISCUSSION THREAD

**Discussion by xoidex** [3kh0/ext-remover#1005] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE]
unenrollment exploit

a friend's classmate somehow just pressed the lock button to sign out and found a "add person" button
and added a person
and somehow unenrolled
they don't know what happened either

help
i am so confused

**Comment by AshtonDavies** [comment]
On some past versions, there has been a bugged add person button on the sign-in screen that is clickable for a second and disappears.

**Reply by xoidex** [reply]
they were on version 121 though

**Reply by xoidex** [reply]
also i'm pretty sure you can recreate the sign in screen thing by pressing esc, i use it to play dino game when there's no internet (also v121)

**Reply by v4pl** [reply]
i have no clue what your talking about be more detailed

**Reply by AshtonDavies** [reply]
> i have no clue what your talking about be more detailed

He can refresh his school's custom sign-in page by pressing `ESC`, because it triggers the back button to a screen that is skipped.

The sign-in screen does not inherit the dino easter egg policy from your account, allowing it to be played that way.

**Reply by xoidex** [reply]
hey just curious, but what screen is skipped?

**Reply by AshtonDavies** [reply]
> hey just curious, but what screen is skipped?

Probably the one informing that it's a managed Chromebook or the default welcome screen.

**Reply by xoidex** [reply]
so theoretically, if someone found a way to not load the managed login page, they would be able to unenroll?

**Comment by ghost** [comment]
TLDR; happened randomly and are wondering how easy to recreate?
Saw this happen so here is some more details: 
It was on an acer spin 511 (octopus board) running v122. She held down the lock button and instead of just the usual buttons like sign out, shut down, and the button to open the control panel, there was a button that read: Add another user or google account (something like that) and then the classmate proceeded to add another user to the Chromebook, in effect completely unenrolling it. With the school's strict tech department there is no way this would ever happen so that's why me and @xoidex were wondering why it might've happened and if it is possible to easily recreate.

**Reply by xoidex** [reply]
nope running v121

**Reply by Quartinal** [reply]
> unenrollment exploit
> 
> a friend's classmate somehow just pressed the lock button to sign out and found a "add person" button and added a person and somehow unenrolled they don't know what happened either
> 
> help i am so confused

@xoidex, me and my friend @PrestonT500 have been looking at this hack (and have been wreaking havoc on the sign-in screen) since early December 2023. I'm glad a discussion popped up for this (I thought this would pop up earlier 😅)! 

Also, another possible way to play the Chrome Dinosaur game on a school Chromebook is with these instructions @PrestonT500 told me about a couple months back:

- Turn your internet off.

- Open any kiosk app.

- Ideally (you most likely have to), wait for the "Waiting for application window" message to pop up.

- Fairly quickly, close your computer until a small light (usually on the left side) turns from blue to red and then open it.

- You should now see a Dino Game screen!

- Note that this sometimes is permanent and sometimes is temporary (usually depending on how much time it takes for you to perform the hack).

**Reply by xoidex** [reply]
haha

so you're also looking for this exploit?

**Reply by Quartinal** [reply]
Yes, though I haven't found a way to un-enroll with the "add person" button, I do know what you're talking about and I also use it to play the dinosaur game on the sign-in screen.

And after I sent that message to you, I found something interesting that can be rolled out in any kiosk app:

- This time, keep your wifi on/enabled.

- Next, open any kiosk app.

- After that, wait for around 2-4 seconds and close your computer. Then wait for a small light (it's the same one mentioned in my previous comment) to turn from blue to red.

- Then open it and a login screen should show up (of course if you natively have one).

- Enter your credentials.

- Then after you submit your credentials, it should continue to open the kiosk app.

- Interesting, right?

**Reply by xoidex** [reply]
yeah pretty interesting

actually, the exploit i'm talking about might have something to do with a kiosk app but i'm not sure.

**Reply by xoidex** [reply]
(doing your thing)
for a split second i saw an add person button, gonna try to click on it

**Reply by xoidex** [reply]
ok it didn't work, but here are my findings

when i signed in it sent my to skiovox thing, and then hit me with the "multiple sign in" page
also attempting to press esc again will send you to the normal sign in screen
and the add person thing isn't pressable like the normal thing where you press esc

**Reply by Quartinal** [reply]
Same for me!

**Reply by Quartinal** [reply]
Anyways, I may have found something via that fullscreen "No internet" screen that you come across when trying to perform Skiovox 120 on older versions:

![thumbnail_20240413_094826](https://github.com/3kh0/ext-remover/assets/148902345/0e326b33-cb31-4a90-a833-f9b17cd2f3f3)
![thumbnail_20240413_094807](https://github.com/3kh0/ext-remover/assets/148902345/cfba4dd3-54f1-4fd9-b65f-378623843574)

More info in just a second.

**Reply by Quartinal** [reply]
Here are the steps to recreate this (this will remove a kiosk app for around 10 minutes):

- Turn your internet off.

- Open any kiosk app.

- You don't have to press Shift+Alt+S for this to work nor Ctrl+Search+Esc.

- Do the regular sign-in steps for Skiovox 120.

- In the "Multiple sign-in has been disabled" screen press Ctrl+Alt+Z to open ChromeVox, then hold down search and then press O two times in a row.

- You should see a "No internet" full-screen screen.

- Now press Ctrl+Alt+Z, then Search+. (period) and then press the settings icon or spam it to see a laggy browser window (probably with your school account) pop up.

- Enjoy!

**Reply by xoidex** [reply]
ooh cool

btw, i found a way to use tab and enter keys on the esc page
it requires some skill though

if you press esc you get to the page
if you press esc and then tab right after (next 20 milliseconds or so) you can switch
you can do this one more time
if you do this again when you press esc it will send you back to the management text because it has some priority in the menu
if you try hard enough you can get to the apps thing
and if you try even harder you can get to the add person thing (i didn't get here yet)

lmfao i hope the button works

**Reply by xoidex** [reply]
also you have to press enter in the same time frame to get the buttons to work

**Reply by xoidex** [reply]
AHHHH

you have to press esc and then tab 3 times in the same frame to get to the add person button
wish me luck

**Reply by xoidex** [reply]
ok so the strat is as follows:

esc + tab
esc + tab + > + > + enter

**Reply by xoidex** [reply]
most i was able to get to is the app thing
i just can't press 5 keys in a frame 😭

**Reply by Blobby-Boi** [reply]
When I do search + o + o it freezes

**Reply by Quartinal** [reply]
By freeze, do you mean that it permanently freezes (until you sign out)? Because when I do it, it usually freezes for <4 seconds.

**Reply by Blobby-Boi** [reply]
It permanently freezes and it doesnt even let me click sign out so I have to reload+poweroff

**Reply by Quartinal** [reply]
What version are you on?

**Reply by Blobby-Boi** [reply]
122

**Reply by Quartinal** [reply]
Hmm, okay. I'm on 121.

You can also do this in a Wifi-on kiosk session using these steps:

- Open any kiosk app (I did this with NWEA).

- After the kiosk app has loaded, press Ctrl+Alt+Z and then:

- If you don't see the ChromeVox menu, press Search+. and then click the settings button.

- If you see the ChromeVox menu, simply click the settings button.

- Please note that this method only works with ChromeVox on.

**Reply by Blobby-Boi** [reply]
Uhm nothing pops up but chromevox says restore pages

**Reply by Quartinal** [reply]
Could you send me a picture?

**Reply by Blobby-Boi** [reply]
![IMG_0190](https://github.com/3kh0/ext-remover/assets/131911782/ce8a51d9-fd32-4e0b-81f2-25182da31787)

**Reply by Blobby-Boi** [reply]
It’s blank

**Reply by Blobby-Boi** [reply]
![IMG_0191](https://github.com/3kh0/ext-remover/assets/131911782/3e7ed961-f49a-4ddc-a363-73276fd3cb0a)

**Reply by Quartinal** [reply]
Okay, I'm not on version 122 so I don't really know how to help you. When my system gets updated, I'll try to find a workaround or another mid-level exploit like this.

**Reply by Quartinal** [reply]
Or maybe try another kiosk app or a URL-based kiosk app like Bluebook or even NWEA?

**Reply by Blobby-Boi** [reply]
Oh wait I got it somehow

**Reply by Blobby-Boi** [reply]
It says no internet now what

**Reply by Quartinal** [reply]
Do you mean the full-screen no internet screen as shown above when you click the link to show previous replies?

**Reply by Blobby-Boi** [reply]
Yes

**Reply by Quartinal** [reply]
Okay, so press Ctrl+Alt+Z and then Search+. and then click the settings icon in the ChromeVox menu and you should be good to go.

**Reply by Blobby-Boi** [reply]
Nope it didn’t do anything

**Reply by Quartinal** [reply]
Could you tell me the steps you did?

**Reply by Blobby-Boi** [reply]
Idk I turned off WiFi but I still got into the kiosk app somehow

**Reply by Quartinal** [reply]
Okay, I will try to find a workaround once I get upgraded to 122.

**Comment by S-PScripts** [comment]
closed unless xoidex chooses to reopen
